### ISSUE TITLE
CSI version not displayed in log message

### ISSUE THREAD

**Issue by yujunz** [juicedata/juicefs-csi-driver#7]
It would be nice to have it for debugging
```
juicefs-plugin I1017 23:57:20.609725       1 driver.go:29[] Driver: csi.juicefs.com Version:
juicefs-plugin I1017 23:57:20.613619       1 mount_linux.go:174[] Cannot run systemd-run, assuming non-systemd OS
```

**Comment by yujunz** [comment]
Need to include also the packaged `juicefs` cli version

**Comment by yujunz** [comment]
Fixed in #25
